mule-deployer/src/main/java/org/mule/tools/model/anypoint/CloudHubDeployment.java

```diff
@@ -37,7 +37,7 @@
   protected Boolean persistentQueues = false;
 
   @Parameter
-  protected Integer waitBeforeValidation = 2000;
+  protected Integer waitBeforeValidation = 6000;
```

Comment:
Had to increase this because on several tests that I made the previous time wasn't enough to get the new status.